tmt/utils.py

```diff
@@ -185,7 +185,7 @@ def run(self, command, message=None, cwd=None, dry=False, shell=True):
         """
         # Use a generic message if none given, prepare error message
         if not message:
-            line = ' '.join(command) if isinstance(command, list) else command
+            line = ' '.join(command) if isinstance(command, list) or isinstance(command, tuple) else command
```

Comment:
Same as above. Also, please make sure the line <= 79 chars. Thanks.

Reply:
OH, sure :) didn't see this one comming :D